tasks/deploy-promo.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Run deploy promo script
+  command: /Users/{{ work_user }}/{{ work_dir }}/{{ promo_dir }}/sh/update.sh
+  environment:
```

Comment:
try
```yml
args:
      chdir: /User/...../
      executable: /bin/bash
```

Reply:
[WARNING]: As of Ansible 2.4, the parameter 'executable' is no longer supported with the 'command' module. Not using '/bin/bash'.